service/app/commands/handler_remove_from_ban_list.go

```diff
@@ -0,0 +1,48 @@
+package commands
+
+import (
+	"github.com/boreq/errors"
+	"github.com/planetary-social/scuttlego/service/domain/bans"
+)
+
+type RemoveFromBanList struct {
+	hash bans.Hash
+}
+
+func NewRemoveFromBanList(hash bans.Hash) (RemoveFromBanList, error) {
+	if hash.IsZero() {
+		return RemoveFromBanList{}, errors.New("zero value of hash")
+	}
+	return RemoveFromBanList{hash: hash}, nil
+}
+
+func (c RemoveFromBanList) IsZero() bool {
+	return c.hash.IsZero()
+}
+
+type RemoveFromBanListHandler struct {
+	transaction TransactionProvider
+}
+
+func NewRemoveFromBanListHandler(transaction TransactionProvider) *RemoveFromBanListHandler {
+	return &RemoveFromBanListHandler{
+		transaction: transaction,
+	}
+}
+
+func (h *RemoveFromBanListHandler) Handle(cmd RemoveFromBanList) error {
+	if cmd.IsZero() {
+		return errors.New("zero value of command")
+	}
+
+	if err := h.transaction.Transact(func(adapters Adapters) error {
+		if err := adapters.BanList.Remove(cmd.hash); err != nil {
+			return errors.Wrap(err, "could not remove the hash from the ban list")
+		}
+		return nil
+	}); err != nil {
+		return errors.Wrap(err, "transaction failed")
+	}
+
+	return errors.New("not implemented")
```

Comment:
Is that supposed to be left here? Seems like it's implemented, isn't it?

Reply:
Yeah that is what I get for no application level tests. I want to go back and add them but there was no way to easily write them when this PR was being created. I added wire setup that creates all the dependencies for application level tests in one of the recent PRs.

I will fix this in a sec and push the changes.

Reply:
[cfa8f65](https://github.com/planetary-social/scuttlego/pull/64/commits/cfa8f65517111d23c1068a8afc7e5a714c0d01b8)